### ISSUE TITLE
need identifier in endpoint so that by seeing the url itself can understand this is belongs to some group

### ISSUE THREAD

**Issue by Manjunath-Harish** [julien-lafont/Mocky#114]
example currently getting something like this
https://run.mocky.io/v3/fcb8caf1-b046-4f4d-bcdf-47acc1558b17
expecting
https://run.mocky.io/v3/fcb8caf1-b046-4f4d-bcdf-47acc1558b17/identifier

**Comment by mudiadamz** [comment]
Yes this is very important rather than random uuid, so we don't need to change those uuids to our real endpoint later on, just chage the prefix.
